webapp/templates/privacy_statement.html

```diff
@@ -5,11 +5,27 @@
         <p>
-            VOKO Utrecht is een collectief van vrijwilligers. Om als collectief gezamenlijk te kunnen functioneren, verzamelen en gebruiken de volgende gegevens van leden: voor- en achternaam, e-mailadres en telefoonnummer. We gebruiken deze gegevens voor de communicatie over en coördinatie van het collectief en voor de administratie van de bestellingen.
+            VOKO Utrecht is een collectief van vrijwilligers. Om als collectief gezamenlijk te kunnen functioneren, verzamelen en gebruiken we de volgende gegevens van leden:
+        </p>
+        <ul>
+            <li>voor- en achternaam,</li>
+            <li>e-mailadres,</li>
+            <li>telefoonnummer,</li>
+            <li>of het lid een rijbewijs heeft,</li>
+            <li>voor welke werkgroepen een lid zich het liefst zou willen inzetten</li>
+            <li>welke dagdelen een lid beschikbaar zou zijn voor VOKO activiteiten,</li>
+            <li>beschikking over een voetuig die gebruikt kan worden voor transport,</li>
+            <li>eventuele talenten die van pas kunnen komen binnen de VOKO.</li>
+        </ul>
+        <p>
+            We gebruiken deze gegevens voor de communicatie over en coördinatie van het collectief, voor het indelen van leden in werkgroepen voor het organiseren van transport en voor de administratie van de bestellingen.
         </p>
         <h2>Gegevens in het bestelsysteem</h2>
         <p>
-            De contact- en bestelgegevens worden bewaard in een eigen bestelsysteem, hiertoe hebben alleen geautoriseerde leden toegang. Deze gegevens worden gebruikt bij de communicatie over het functioneren van het collectief en de bestelronden en voor de (financiële) administratie.
+            De contact- en bestelgegevens worden bewaard in een eigen bestelsysteem, hiertoe hebben alleen geautoriseerde leden toegang. Deze gegevens worden gebruikt bij de communicatie over het functioneren van het collectief, de bestelronden, de transport en voor de (financiële) administratie.
+        </p>
+        <p>
+            Om de bestelde producten van de levanciers te transporteren naar het ophaalpunt worden transport roosters opgesteld. Leden krijgen alleen inzicht in de ritten waar ze zelf betrokken bij zijn. Bij deze ritten krijgen ze de contact gegevens van het lid waar ze mee rijden, transport coordinatoren en van de levanciers waar ze producten gaan ophalen. Ritten uit het verleden zijn ontoegankelijk. Leden die transport coordineren of financiele administratie doen hebben wel toegang tot het volledige transport rooster.
         </p>
         <p>
             Om de bestellingen klaar te zetten tijdens uitdeelavonden worden overzichten per besteller uitgedraaid. De contactgegevens van bestellers staan op deze overzichten en kunnen gebruikt worden om laatkomers te contacteren. Bestellers zijn zelf verantwoordelijk voor het meenemen van het persoonlijke besteloverzicht, overzichten die aan het einde van de uitdeelavond overblijven worden weggegooid.
```

Comment:
Een paar suggesties, en ik weet dat ik een grammar nazi ben ;)

> of het lid een rijbewijs heeft

`beschikking over een rijbewijs`

> voor welke werkgroepen een lid zich het liefst zou willen inzetten

`voorkeurswerkgroepen`

> welke dagdelen een lid beschikbaar zou zijn voor VOKO activiteiten,

`beschikbaarheid voor VOKO activiteiten per dagdeel`

> beschikking over een voetuig die gebruikt kan worden voor transport

`beschikking over een voertuig dat gebruikt kan worden voor transport`

> eventuele talenten die van pas kunnen komen binnen de VOKO

Ik zou "eventuele" weglaten;
de VOKO --> het VOKO

> We gebruiken deze gegevens voor de communicatie over en coördinatie van het collectief, voor het indelen van leden in werkgroepen voor het organiseren van transport en voor de administratie van de bestellingen.

`We gebruiken deze gegevens voor de communicatie over- en coördinatie van het collectief, voor het indelen van leden in werkgroepen, voor het organiseren van transport en voor de administratie van de bestellingen.`

> de transport (regel 25)

het transport

Dit stukje heb ik een beetje herschreven:

> Om de bestelde producten van de levanciers te transporteren naar het ophaalpunt worden transportroosters opgesteld. Leden hebben uitsluitend inzicht in ritten waarbij zij zelf betrokken zijn. Van deze ritten worden de contactgegevens van ingeroosterde leden en transportcoördinatoren met elkaar gedeeld. Daarnaast worden contactgegevens van betreffende levanciers gedeeld met deze leden. Ritten uit het verleden zijn ontoegankelijk. Leden die het transport coördineren of de financiële administratie bijhouden, hebben toegang tot het volledige transportrooster.

Reply:
Goede verbeteringen!